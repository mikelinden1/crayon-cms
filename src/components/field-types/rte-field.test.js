import RteField from './rte-field';

const props = {
    name: 'test',
    value: '<p>test</p>',
    onChange: () => {},
    disabled: false
};

it('RteField matches snapshot', () => {
    const wrapper = shallow(<RteField {...props} />);
    expect(wrapper).toMatchSnapshot();
});
