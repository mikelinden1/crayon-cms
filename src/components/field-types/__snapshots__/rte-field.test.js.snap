// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RteField matches snapshot 1`] = `
<CKEditor
  activeClass=""
  config={
    Object {
      "allowedContent": true,
      "filebrowserUploadUrl": "http://localhost/php-crud-api/ck-upload?uploadFullPath=http://localhost/uploads&uploadPath=../uploads",
    }
  }
  content="<p>test</p>"
  events={
    Object {
      "afterPaste": [Function],
      "blur": [Function],
      "change": [Function],
      "key": [Function],
    }
  }
  isScriptLoaded={false}
  scriptUrl="https://cdn.ckeditor.com/4.6.2/standard/ckeditor.js"
/>
`;
